lego/apps/users/fixtures/test_abakus_groups.yaml

```
@@ -326,7 +326,7 @@
     type: annen
     text: ''
     permissions: '["/sudo/admin/meetings/create/", "/sudo/admin/meetinginvitations/create/",
-      "/sudo/admin/registrations/create/", "/sudo/admin/events/payment/"]'
+      "/sudo/admin/registrations/create/", "/sudo/admin/events/payment/", "/sudo/admin/submissions/create"]'
```

PR is long (ish)
me is tired
I skimmed these files
pls add end-slash


```suggestion
      "/sudo/admin/registrations/create/", "/sudo/admin/events/payment/", "/sudo/admin/submissions/create/"]'
```